connector/doc/guides/bootstrap_connector.rst

```diff
@@ -208,6 +211,23 @@ created. I propose to create a helper too in
         return checkpoint.add_checkpoint(session, model_name, record_id,
                                          'coffee.backend', backend_id)
 
+Message checkpoint
+------------------
+
+When you need to show a plain text message to a user
+(for instance: an import job did not succeeded),
```

Comment:
But go and search in the failed job should not be an issue, usually you should have *no* failed jobs, as a user you have to monitor them. Besides, you can subscribe to the job failures to receive notifications. The use case I saw with you in the prestashop connector was more like the one I described: you are importing a product, but you couldn't import the category associated to the product. Never mind, we mark the job as success and add a checkpoint saying that its category could not be imported. I didn't realized before now that in this case we have a model to attach to and that's the product. Anyway, I'm sure there are use cases where adding a message without a record is needed.